### PR TITLE
Add basic coverage to RailsStats::CodeStatistics#to_s

### PR DESCRIPTION
**Pull request by etagwerker**
Hey @bleonard, 

This PR adds minitest to the project and it adds a simple scenario for `RailsStats::CodeStatistics#to_s`

I added a Rails 6.0 application in test/dummy to make it easier to test `rails_stats`. I didn't want to add a lot of tests before getting your feedback. Do you think this is a step in the right direction? 

This is what it should look like in your test environment:

Screen Shot 2020-10-06 at 11 48 25 PM

Please let me know. 

Thanks!

### CODE REVIEW

**Comment by bleonard** [comment]
That's certainly a bunch of files, but guess they are all in the test directory.
Feel free to give it a shot if you want. I didn't even know people were using this project :-)

**Comment by etagwerker** [comment]
@bleonard Cool, thanks! Do you think this is ready to be merged? I just solved the conflict. 😄

**Comment by etagwerker** [comment]
@bleonard One more thing, could you enable Travis CI for this project? Or do you prefer checks with GitHub Actions?

File: Rakefile [hunk 1]
```diff
@@ -1 +1,12 @@
-require "rails_stats"
```
Comment (arielj) [inline Rakefile:1]: Removing this lines makes the rake command unable to find the `stats` task.

File: test/test_helper.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require "byebug"
+
+require "minitest/autorun"
+require "minitest/pride"
+require "minitest/around/spec"
+
+require "rails_stats/all"
```
Comment (arielj) [inline test/test_helper.rb:9]: Just nit picking here, but it's warning about a missing trailing new line

**Review by arielj — CHANGES_REQUESTED** [review]
It would be nice to have a section in the README explaining how to run the tests of the gem.

**Comment by etagwerker** [comment]
@arielj Thanks for review this PR! Just addressed your comments 👍

**Review by arielj — APPROVED** [review]
Looks good!

**Pull request merged**